rules/gb-model.smk

```diff
@@ -56,3 +54,24 @@ rule manual_region_merger:
         "../envs/environment.yaml"
     script:
         "../scripts/gb-model/manual_region_merger.py"
+
+
+rule compose_network:
+    input:
+        unpack(input_profile_tech),
```

Comment:
name this so you can access that named file list in the script (rather than relying on the filename starting with `profile_`).

Reply:
unfortunately, snakemake does not like named inputs with `unpack`. so sticking to the old behavior, but made it leaner in the code.